k8s/plugin/README.md

```diff
@@ -205,57 +205,44 @@ Volume 0c08667c-8b59-4d11-9192-b54e27e0ce0f Scaled Successfully 🚀
-            Kubernetes namespace of mayastor service [default: mayastor]
-
-    -o, --output <OUTPUT>
-            The Output, viz yaml, json [default: none]
-
-    -r, --rest <REST>
-            The rest endpoint to connect to
-
-    -s, --since <SINCE>
-            Period states to collect all logs from last specified duration [default: 24h]
-
-    -t, --timeout <TIMEOUT>
-            Specifies the timeout value to interact with other modules of system [default: 10s]
-
-SUBCOMMANDS:
-    help       Print this message or the help of the given subcommand(s)
-    node       Collects information about particular node matching to given node ID
-    nodes      Collects information about all nodes
-    pool       Collects information about particular pool and its descendants matching to given
-                   pool ID
-    pools      Collects information about all pools and its descendants (nodes)
-    system     Collects entire system information
-    volume     Collects information about particular volume and its descendants matching to
-                   given volume ID
-    volumes    Collects information about all volumes and its descendants (replicas/pools/nodes)
+Usage: kubectl-mayastor dump [OPTIONS] <COMMAND>
+
+Commands:
+  system   Collects entire system information
+  volumes  Collects information about all volumes and its descendants (replicas/pools/nodes)
+  volume   Collects information about particular volume and its descendants matching to given volume ID
+  pools    Collects information about all pools and its descendants (nodes)
+  pool     Collects information about particular pool and its descendants matching to given pool ID
+  nodes    Collects information about all nodes
+  node     Collects information about particular node matching to given node ID
+  etcd     Collects information from etcd
+  help     Print this message or the help of the given subcommand(s)
+
+Options:
+  -r, --rest <REST>
+          The rest endpoint to connect to
+  -t, --timeout <TIMEOUT>
+          Specifies the timeout value to interact with other modules of system [default: 10s]
+  -k, --kube-config-path <KUBE_CONFIG_PATH>
+          Path to kubeconfig file
+  -s, --since <SINCE>
+          Period states to collect all logs from last specified duration [default: 24h]
+  -l, --loki-endpoint <LOKI_ENDPOINT>
+          Endpoint of LOKI service, if left empty then it will try to parse endpoint from Loki service(K8s service resource), if the tool is unable to parse from service then logs will be collected using Kube-apiserver
+  -e, --etcd-endpoint <ETCD_ENDPOINT>
+          Endpoint of ETCD service, if left empty then will be parsed from the internal service name
+  -d, --output-directory-path <OUTPUT_DIRECTORY_PATH>
+          Output directory path to store archive file [default: ./]
+  -n, --namespace <NAMESPACE>
+          Kubernetes namespace of mayastor service, defaults to mayastor [default: mayastor]
+  -c, --current-logs
```

Comment:
Since this includes both current and historical logs, maybe, this should be "--include-current-logs"

Reply:
Yeah, that sounds good. Will make this change.